README.md

```diff
@@ -1,24 +1,53 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Blog app📝
+
+- This is a simple app made using Ruby on Rails where you can keep track of your expenses. You can see a video demo [here](https://www.loom.com/share/22519db3d9b54aa3a191e9e224ab2f39)
+
+## Built With
+
+- Ruby
+- Ruby on Rails
 
 Things you may want to cover:
 
-* Ruby version
+- Ruby version
+
+* 3.1.2
+
+- Configuration
+
+* git clone `https://github.com/Thi-Ponce/blog-app.git`
+
+* change to the directory using `cd blog-app`
+
+* open with your preffered code editor.
+
+* remember to use commands to install all necessary gems and requirements such as: `bundle install`, `rails db:create`, `rails db:migrate`, and `yarn install` or `npm install`
+
+## Authors:
+
+👤 **Thiago Allan Porro Ponce**
+
+- GitHub: [@Thi-Ponce](https://github.com/Thi-Ponce)
+- Twitter: [@Thi_ponce](https://twitter.com/Thi_ponce)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/thiago-ponce)
+
+## 🤝 Contributing
 
-* System dependencies
+Contributions, issues, and feature requests are welcome!
 
-* Configuration
+## Show your support
 
-* Database creation
+Give a ⭐️ if you like this project!
 
-* Database initialization
+## Acknowledgments
 
-* How to run the test suite
+- Microverse: [microverse community](https://github.com/microverseinc)
+- All the friends that helped me along the way.
```

Comment:
- [ ] In the acknowledgement section of this README.md file, kindly give credits to the designer of the template you used for this project. The name of the designer is **_Gregoire Vella_** on Behance.